pkg/kv/kvserver/protectedts/ptpb/protectedts.proto

```diff
@@ -86,6 +87,18 @@ message Metadata {
    uint64 total_bytes = 4;
 }
 
+message SchemaObjectsTarget {
```

Comment:
These could also be nested messages within `message Record`, I assume they'll not see any independent use. If I'm understanding correctly, these IDs can _only_ correspond to databases and tables, right? 

s/in this target/being protected

Reply:
done.